app/lint-baseline.xml

```diff
@@ -1,6 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <issues format="5" by="lint 3.5.0" client="gradle" variant="all" version="3.5.0">
 
+    <issue
```

Comment:
Is there a better way to manage lint warnings/errors? We have adopted the habit of just updating the baseline each time we have an error which to my understanding is not how the baseline file is intended to be used.

Reply:
No, I think what you have in place is great! I do my best to fix the lint error before simply adding it to the baseline, but in this case I believe it is a false positive. I'll try out your suggestion below to see if that resolves it.